examples/simple_classifier_cifar10/start_with_hpo.py

```diff
@@ -20,8 +20,7 @@
 }
 
 if __name__ == "__main__":
-
-    AWS_ID = "0123456789"  # use your AWS account id here
+    AWS_ID = boto3.client("sts").get_caller_identity().get("Account")
     AWS_REGION = "us-west-2"  # use your AWS preferred region here
```

Comment:
It would be great to change region as well:
```
AWS_REGION = boto3.session.Session().region_name
```

Reply:
This is subjective and not as critical I believe, I think it is best done by the core contributor if they prefer this other way.

Reply:
I'm fine with it @hfurkanbozkurt @geoalgo feel free to add it if you want to

Reply:
Personally, I think having the region in code is fine for this example and may even be better as it shows how you can change the region from the default one easily (which you need to do if you want to have some instances ATM, for instance g5 are only in us-east).